receiver/k8sclusterreceiver/internal/collection/clusteresourcequotas.go

```diff
@@ -0,0 +1,124 @@
+// Copyright 2020 OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package collection
+
+import (
+	"strings"
+
+	metricspb "github.com/census-instrumentation/opencensus-proto/gen-go/metrics/v1"
+	resourcepb "github.com/census-instrumentation/opencensus-proto/gen-go/resource/v1"
+	quotav1 "github.com/openshift/api/quota/v1"
+	"go.opentelemetry.io/collector/translator/conventions"
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/k8sclusterreceiver/internal/utils"
+)
+
+var clusterResourceQuotaLimitMetric = &metricspb.MetricDescriptor{
```

Comment:
At the time this receiver was written metadata.yaml and pdata weren't around. Would it be possible to add net-new stuff using the new ways so there's less code that has to be translated in the future?

Reply:
Happy to have an issue assigned to me and do it in a separate PR.

Reply:
#4367